plasmapy/physics/quantum.py

```diff
@@ -99,14 +98,15 @@ def deBroglie_wavelength(V, particle):
 
         lambda_dBr = np.ones(V.shape) * np.inf * units.m
         indices = V.value != 0
-        lambda_dBr[indices] = h / (m*V[indices]*Lorentz_factor(V[indices]))
+        lambda_dBr[indices] = h / \
```

Comment:
Remove the slash for continuing the line. It's quite bad practice. Just enclose the entire statement in parentheses.

Reply:
Okay, that was me. Well, it was me, autopep8 and a lack of dilligence.